plot.py

```diff
@@ -562,12 +564,13 @@ def _update_plot(self):
                                 ax=self.ax, title=self.title)
             self.display.set_limits(xlim=(self.limits['xmin'], self.limits['xmax']),\
                                     ylim=(self.limits['ymin'], self.limits['ymax']),\
-                                    ax=self.ax)
+                                    ax=self.ax, fig=self.fig)
```

Comment:
This is fine, I did correct one instance where it was passed to the wrong routine.